### PR TITLE
Improve feature flags CLI

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: scripts/completion/cozy-stack.bash [hunk 1]
```diff
@@ -1368,8 +1368,16 @@ _cozy-stack_features()
 
     commands=()
     commands+=("context")
+    if [[ -z "${BASH_VERSION}" || "${BASH_VERSINFO[0]}" -gt 3 ]]; then
+        command_aliases+=("context")
+        aliashash["context"]="context"
```
Comment (sblaisot) [inline scripts/completion/cozy-stack.bash]: `="ratio"` ?

File: scripts/completion/cozy-stack.bash [hunk 2]
```diff
@@ -1368,8 +1368,16 @@ _cozy-stack_features()
 
     commands=()
     commands+=("context")
```
Comment (sblaisot) [inline scripts/completion/cozy-stack.bash]: `commands+=("ratio")` ?

File: cmd/feature.go [hunk 3]
```diff
@@ -164,17 +165,18 @@ All the sets can be removed by setting an empty list ('').
 	},
 }
 
-var featureContextCmd = &cobra.Command{
-	Use:   "context <context-name>",
-	Short: `Display and update the feature flags for a context`,
+var featureRatioCmd = &cobra.Command{
+	Use:     "context <context-name>",
```
Comment (sblaisot) [inline cmd/feature.go]: s/context/ratio/ ?

Comment (nono) [reply cmd/feature.go]: Good catch :+1:

**Pull request merged**